api/service/legacysync/epoch_syncing.go

```diff
@@ -101,13 +101,25 @@ func syncLoop(bc core.BlockChain, syncConfig *SyncConfig) (timeout int) {
 		return 10
 	}
 
+	utils.Logger().Info().
+		Bool("isBeacon", isBeacon).
+		Uint32("ShardID", bc.ShardID()).
+		Uint64("otherEpoch", shard.Schedule.CalcEpochNumber(maxHeight).Uint64()).
+		Uint64("currentEpoch", bc.CurrentBlock().Epoch().Uint64()).
+		Int("peers count", syncConfig.PeersCount()).
+		Msg("[EPOCHSYNC] Node is OUT OF SYNC, it's trying to get fully synchronized")
+
 	for {
 		curEpoch := bc.CurrentBlock().Epoch().Uint64()
 		otherEpoch := shard.Schedule.CalcEpochNumber(maxHeight).Uint64()
 		if otherEpoch == curEpoch+1 {
 			utils.Logger().Info().
-				Msgf("[EPOCHSYNC] Node is now IN SYNC! (isBeacon: %t, ShardID: %d, otherEpoch: %d, currentEpoch: %d, peersCount: %d)",
-					isBeacon, bc.ShardID(), otherEpoch, curEpoch, syncConfig.PeersCount())
+				Bool("isBeacon", isBeacon).
```

Comment:
@GheisMohammadi i'm doing this intentionally, because you have to scroll from left to right to get info about what you need. When we keep all data in `Msgf` you have to check only msg string. Your case is good for automated search, but, as i know, we all do it by hands. If you use some automation for this pls tell me, because i don't know about this. Keep it separated just for the future doesn't look reasonalbe

Reply:
I'll resolve this conversation for now. Feel free to continue the discussion though